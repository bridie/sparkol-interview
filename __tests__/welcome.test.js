import React from 'react';
import renderer from 'react-test-renderer';

import Welcome from '../src/js/components/welcome';

it('renders correctly', () => {
  Welcome.generateGreeting = jest.fn(() => 'Hola');
  Welcome.getUserName = jest.fn(() => 'Will Shakes')

  const tree = renderer.create(<Welcome />).toJSON();
  expect(tree).toMatchSnapshot();
});
