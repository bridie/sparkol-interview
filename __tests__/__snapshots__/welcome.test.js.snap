// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Hola, Will Shakes
  </h1>
  <button
    className="c-button"
    onClick={[Function]}
    type="button"
  >
    Logout
  </button>
</div>
`;
